eloquentjs/11_3_real_promises/src/gaussian.js

```diff
@@ -0,0 +1,6 @@
+function gaussian(x, variance, center, scale = 1) {
```

Comment:
I'd just include this in the generateLogs file. Lots of files means lots of context shifts.